### PR TITLE
MAT-6823 QDM Improvement Notation text field

### PR DESCRIPTION
**Pull request by adongare**
## MADiE PR Template

Jira Ticket: [MAT-6823](https://jira.cms.gov/browse/MAT-6823)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e. Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/models/measure/QdmMeasure.java [hunk 1]
```diff
@@ -27,4 +27,5 @@ public class QdmMeasure extends Measure {
 	private String rateAggregation;
 	private String improvementNotation;
 	private String improvementNotationOther;
```
Comment (adongare) [inline src/main/java/gov/cms/madie/models/measure/QdmMeasure.java]: moving this to separate model. cleanup PR to follow.

**Review by sb-cecilialiu — APPROVED** [review]

File: src/main/java/gov/cms/madie/models/measure/QdmMeasure.java [hunk 2]
```diff
@@ -26,5 +26,5 @@ public class QdmMeasure extends Measure {
 	private boolean patientBasis = true;
 	private String rateAggregation;
 	private String improvementNotation;
-	private String improvementNotationOther;
+	private String improvementNotationDescription;
```
Comment (RohitKandimalla) [inline src/main/java/gov/cms/madie/models/measure/QdmMeasure.java:29]: Wouldn't we need this description for FHIR Measures ?

Comment (RohitKandimalla) [reply src/main/java/gov/cms/madie/models/measure/QdmMeasure.java:29]: Nvm FHIR has it at Group level, I just remembered

Comment (adongare) [reply src/main/java/gov/cms/madie/models/measure/QdmMeasure.java:29]: yeah right
